server/channels/api4/import.go

```diff
@@ -31,3 +33,24 @@ func listImports(c *Context, w http.ResponseWriter, r *http.Request) {
 		c.Logger.Warn("Error writing imports", mlog.Err(err))
 	}
 }
+
+func deleteImport(c *Context, w http.ResponseWriter, r *http.Request) {
+	auditRec := c.MakeAuditRecord("deleteImport", audit.Fail)
+	defer c.LogAuditRec(auditRec)
+	auditRec.AddMeta("import_name", c.Params.ImportName)
+
+	if !c.IsSystemAdmin() {
```

Comment:
Nit pick: most of the code checks the permission directly:
```suggestion
	if !c.App.SessionHasPermissionTo(*c.AppContext.Session(), model.PermissionManageSystem)  {
```